app/src/main/java/dev/andrylat/raqimbek/bankingutils/CardValidator.java

```diff
@@ -1,23 +1,32 @@
 package dev.andrylat.raqimbek.bankingutils;
```

Comment:
There is no split by packages.

Every piece of logic should be in separate packages. Validators - on package. Dialogs - second one, etc.

The best if each feature is in own package.